examples/things_advanced.py

```diff
@@ -45,48 +43,53 @@ def __call__(self, req, resp, engine):
 
 
 class AuthMiddleware:
-
     def process_request(self, req, resp):
         token = req.get_header('Authorization')
         account_id = req.get_header('Account-ID')
 
         challenges = ['Token type="Fernet"']
 
         if token is None:
-            description = ('Please provide an auth token '
-                           'as part of the request.')
+            description = 'Please provide an auth token ' 'as part of the request.'
```

Comment:
These changes look odd. We would probably need to carefully review them manually if we want to keep the same high standard of Falcon's source code.

Reply:
reaching for `' '` finds 92 results. I think it would be manageable

Reply:
Wondering if `black` is making any progress in this area? :thinking: 
IIRC there were discussions to handle this out of the box, but I haven't checked what the status is.

Reply:
Not much. It now remosed leading and trailing spaces in triple comments where it makes sense.

But thsi caae needs to be handled by hand.

Reply:
I think the intention is to handle all strings: https://github.com/psf/black/issues/2188

But it seems there are still some issues with it.

Reply:
ah, sqlalchemy makes black file, ahah :)

Reply:
tried running it. not particularly convinced:

see the diff https://gist.github.com/CaselIT/2e509ce7b7425700a87f190669baeb1c